apps/usgsearthquakes/usgs_earthquakes.star

```diff
@@ -60,7 +69,8 @@ DEFAULT_LOCATION = """
 }
 """
 DEFAULT_MAGNITUDE = "3"
-DEFAULT_RADIUS = "0"
+DEFAULT_RADIUS = "250"
```

Comment:
My preference would be to keep the default radius at zero. While 250 makes sense for the area I live in, for many people this results in nothing to show for most of the time.

Reply:
Will update and commit.